src/XmlWriter.php

```diff
@@ -235,6 +255,16 @@ public function setXmlVersion(string $xmlVersion): XmlWriter
         return $this;
     }
 
+    /**
+     * Set the XML standalone
+     */
+    public function setXmlStandalone(bool $xmlStandalone): XmlWriter
```

Comment:
```suggestion
    public function setXmlStandalone(bool $xmlStandalone): static
```

Reply:
would you like `setXmlEncoding` and `setXmlVersion` also marked with static for the return type? They are currently `XmlWriter`

Reply:
Change made, also made the change for `setXmlEncoding` and `setXmlVersion`